test/helpers/block.ts

```diff
@@ -271,33 +271,6 @@ export const verifyBlockFees = async (
             }
           }
         }
-        // Then search for Deposit event from treasury
```

Comment:
should we be checking these from other event instead? say `balances.Deposit` on the treasury address or something?